### DISCUSSION TITLE
lighthouse-badger-easy Input required and not supplied: token

### DISCUSSION THREAD

**Discussion by Salih2000** [alshedivat/al-folio#2633] [Q&A] [answered]
Hey I added my token on secrets as named on the code but I still get this error, Can you pls help me what's wrong here ?
![image](https://github.com/user-attachments/assets/d5d4819d-f7a9-4f73-83cc-cc8b962dc4ca)

**Comment by george-gca** [comment]
No idea why considering that the token has the right properties. Have you checked their repo to see if anything changed recently, or if there are any issues about this?

**Reply by Salih2000** [reply]
![image](https://github.com/user-attachments/assets/060347e6-407a-4151-913b-7c3c360a44ac)
Well while creating my token, I configured the settings accordingly to this screenshot . I also conducted thorough research to find any up-to-date solutions for this issue. Unfortunately, I only found topics that were unrelated or too old to my specific problem.

**Reply by george-gca** [reply]
Follow the steps in our [FAQ](https://github.com/alshedivat/al-folio/blob/master/FAQ.md#when-i-manually-run-the-lighthouse-badger-workflow-it-fails-with-error-input-required-and-not-supplied-token-how-do-i-fix-that) about that. I recommend you creating a [PAT (fine-grained)](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-fine-grained-personal-access-token) with the scopes and permissions suggested for the [lighthouse-badger-easy.yml](https://github.com/MyActionWay/lighthouse-badger-workflows/blob/main/.github/workflows/lighthouse-badger-easy.yml).

**Comment by matinator** [comment]
I did as PAT as required, but still it doesn't find it in the github action, any news on how to possibly fix it?

**Reply by Salih2000** [reply]
Unfortunately not, I couldn't find any direct solution for this problem, so I gave up to use Alfolio and move on a different theme, but I'm still seeking for the what is causing this error.

**Reply by matinator** [reply]
Thank you for your quick reply. I will try to comeback to this theme when it works.

**Reply by Salih2000** [reply]
I will be very happy if you solve that out and share with the community.

**Reply by matinator** [reply]
I will happily do that if I am able to find a solution, but it depends mainly to find a workaround of github and that can be quite difficult.

**Reply by matinator** [reply, marked as answer]
Solved it in the end, it was easier than I thought, I only had to change the worflow at line 53 of lighthouse-badger.yml with:

`token: ${{ secrets.LIGHTHOUSE_BADGER_TOKEN }}`

instead of the original one

**Reply by Salih2000** [reply]
this change solved my problem. My page is live now, thank you very much for your help, Grazie!
![image](https://github.com/user-attachments/assets/91bb2c45-7595-4016-b883-f355b452b305)

**Reply by george-gca** [reply]
This is weird, because it doesn't throw error on the main repo. But indeed this line uses a param that is not defined. I will updated it accordingly.

**Reply by george-gca** [reply]
Also note that this workflow is not needed to deploy the site, only to evaluate its speed and compliance to some standards.

**Reply by Salih2000** [reply]
![image](https://github.com/user-attachments/assets/3d009c4a-0d95-4666-84cf-28f80b15345e)
On  https://github.com/alshedivat/al-folio/blob/master/INSTALL.md page, there is a video manual. If you open it, you can observe the error at the 2:01 mark in the timeline. Thank you for your contribution.

**Reply by george-gca** [reply]
The action will fail because the secret token is not created, but after that it works fine, that is what I meant.
